### ISSUE TITLE
[BE Module Colour] - Wich colour to choose for the backend module similarity bar %

### ISSUE THREAD

**Issue by friteuseb** [friteuseb/semantic_suggestion#5]
as we're talking about the Top 5 Most Similar Page Pairs  it's tempting to choose green when 2 pages are very similar, but it's not that good for SEO to have very similar pages. 

What is a good % of similarities ?

**Comment by dkd-dobberkau** [comment]
I'm not sure if this fits, but this could be an idea.

![Screenshot 2024-09-07 at 17-07-03 Claude](https://github.com/user-attachments/assets/54ded379-285b-4ea4-bdbb-30d97827b42f)

**Comment by friteuseb** [comment]
Yes i like this colour code olivier, i like also the Idea of a table but it could be for the best  match only ?
